testsuite/pangu_lib/testnet_commands/transaction_emitter.py

```diff
@@ -0,0 +1,95 @@
+from .create_testnet import SystemContext
+import pangu_lib.util as util
+from kubernetes import client
+from test_framework.logging import log
+from typing import List
+import pangu_lib.util as util
+
+import random
+import string
+import time
+
+
+def transaction_emitter_main(
+    testnet_name: str, args: List[str], system_context: SystemContext, timeout: int = 30, ask_for_delete: bool = True
+):
+    #
+    # Create command array
+    command_array = ["aptos-transaction-emitter"]
+    command_array.extend(args)
+
+    #
+    # Create pod name
+    random_postfix: str = "".join(
+        random.choices(string.ascii_lowercase + string.digits, k=8)
+    )
+    pod_name = f"{testnet_name}-tx-emitter-{random_postfix}"
+
+    #
+    # Create Pod
+    log.info("Starting transaction emitter...")
+    pod: client.V1Pod = create_transaction_emitter_pod(pod_name, command_array)
+    system_context.kubernetes.create_resource(pod, testnet_name)
+    log.info("Transaction emitter started...")
```

Comment:
Ok, 2 ways to go about dry_run. To have the dry run functionality, the user would need to provide a dry_run bool and workspace str. These 2 ways exist because of how we are piping all options thru to the transaction emitter. 

- The CLI can take this as an arg, in which case the user would need to explicitly provide this info every time (can't be optional) even if they dont want to do a dry run. 
- We can let the user have the illusion of providing an option for the dry-run/workspace, but on the code side we'd traverse the args that are parsed to be sent to the transaction emitter to see if they've sent a --dry_run and --workspace option.

What's your preference? I prefer the first option.

Reply:
Nvm, fixed this.